app/routes.js

```diff
@@ -1,8 +1,10 @@
 // const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
+const { weetController } = require('./controllers/api/v1/weet_controller');
 
 exports.init = app => {
   app.get('/health', healthCheck);
+  app.get('/api/v1/weet', weetController.get);
```

Comment:
Good Job in this card it is not necessary to expose the service in an endpoint but It's very well